docs/android-startup-scenarios.md

```diff
@@ -0,0 +1,35 @@
+
+# Android Startup Test
+
+## Prereqs
+
+- Ensure `dotnet` is installed and available with the `dotnet` command. [Dotnet Download](https://dotnet.microsoft.com/en-us/download) or [Daily Dotnet Download](https://github.com/dotnet/sdk/blob/main/documentation/package-table.md)
```

Comment:
Do we have any requirement regarding minimal .net version?

Reply:
Added a comment on the .NET Core version to have. Which should be any supported version. Although this may not be true depending on if and how supporting startup testing tools are generated.